### PR TITLE
Added borgmatic

### PR DESCRIPTION
**Pull request by BarbzYHOOL**
Added that great general purpose wrapper

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ The preferred way is that you put a link to your own repository here:
 - https://github.com/Alex131089/bbbs (Bash Borg Backup System – wrappers to handle pull-mode operation. Also comes with installer and ssh key provisioning)
 - https://github.com/makeITyourway/borg_wrapper (Another Borg wrapper enables you to handle multiple projects and configs per project)
 - https://github.com/engelant/borg-cygwin (Cygwin based Windows installer creator, with VSS based backup script)
+- https://github.com/witten/borgmatic (Python wrapper - Works with config files)
```
Comment (ThomasWaldmann) [inline README.rst]: oh, that one was missing still?

yeah, it should definitely be on that list but can we have some better description?

"Python wrapper" (borg wrapper?) and "works with config files" doesn't say much about why it is actually useful.

Comment (BarbzYHOOL) [reply README.rst]: i did my best, no idea what to say since I only use borgmatic, I have no reference to compare with

**Comment by rugk** [comment]
Is not borgmatic included in the official borg docs, so tzhat's maybe why you did not include it here?

**Comment by ThomasWaldmann** [comment]
Yeah, in fact we have it in the docs:

https://borgbackup.readthedocs.io/en/stable/resources.html?highlight=borgmatic

Description there:

borgmatic - simple wrapper script for BorgBackup that creates and prunes backups

So, we can either just leave it as is or add it here with same or better description.

@BarbzYHOOL the current description in your PR almost says nothing, so we either need to improve it or leave it.

**Comment by BarbzYHOOL** [comment]
updated even though it doesn't say much either to me lol

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**